src/chatterbox/models/t3/fast_min_p_warper.py

```
@@ -0,0 +1,34 @@
+import torch
+from transformers.generation.logits_process import MinPLogitsWarper
+
+
+class FastMinPLogitsWarper(MinPLogitsWarper):
+    def __init__(self, *args, device="cuda", **kwargs):
+        super().__init__(*args, **kwargs)
+        self.false_tensor = torch.zeros(1, dtype=torch.bool, device=device)
+        self.min_p = torch.tensor(self.min_p, device=device)
+
+    def __call__(
+        self, input_ids: torch.LongTensor, scores: torch.FloatTensor
```

Ruff (0.13.1)

Unused method argument: `input_ids`

(ARG002)